dc-cudami-client/src/main/java/de/digitalcollections/cudami/client/identifiable/entity/CudamiDigitalObjectsClient.java

```diff
@@ -27,6 +30,19 @@ public CudamiDigitalObjectsClient(HttpClient http, String serverUrl, ObjectMappe
     return doGetRequestForObjectList(baseEndpoint + "/reduced", DigitalObject.class);
   }
 
+  public SearchPageResponse<DigitalObject> findAllForParent(DigitalObject parent)
+      throws TechnicalException {
+    if (parent == null) {
+      throw new TechnicalException("Empty parent");
+    }
+
+    SearchPageRequest searchPageRequest = new SearchPageRequest(null, 0, 10000);
```

Comment:
Reicht hier nicht ein normaler `PageRequest`?

Reply:
Im Prinzip ja, aber im `CudamiRestClient` ist die `find`-Methode für einen `PageRequest` als deprecated annotiert.

Eine von beiden Stellen sollten wir dann ändern.

Reply:
Ich würde die Annotation wieder wegnehmen :thinking:

Reply:
Umbau (ging bis in den Controller) ist gepushed